### PR TITLE
TW-937 Doc support for JWT auth

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good!

File: src/pages/docs/sending-requests/authorization.md [hunk 1]
```diff
@@ -129,6 +130,29 @@ Bearer <Your API key>
 
 > If a custom prefix is needed, use an [API Key](#api-key) with a key of __Authorization__.
 
+### JWT bearer
+
+Postman also supports generating JWT bearer tokens to authorize requests. You can enter a payload in an an editor, and JWT tokens are generated and added to the request. In the request __Authorization__ tab, select __JWT Bearer__ from the __Type__ dropdown list.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/authorization.md:135]: You can enter a Payload in `the` editor?

File: src/pages/docs/sending-requests/authorization.md [hunk 2]
```diff
@@ -129,6 +130,29 @@ Bearer <Your API key>
...
+    * **PS** - RSA (RSASSA-PSS) with SHA
+
+* **Secret** - The secret that’s used with the HMAC-SHA algorithm.
+* **Secret Base64 encoded** - If the secret is encoded in the base-64 format.
+
+* **Payload** - Enter the payload data for your JWT token, in JSON format.
+
+In the Advanced configuration section, you can also configure the following items. If you don't configure them, they are generated automatically.
+
+* **Header prefix** - An optional prefix to use at the start of headers.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/authorization.md:152]: Cna we specifically mention that this header prefix is part of the request and not a part of JWT?

File: src/pages/docs/sending-requests/authorization.md [hunk 3]
```diff
@@ -129,6 +130,29 @@ Bearer <Your API key>
...
+
+* **Headers** - Any custom headers you also want to send in the JWT token.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/authorization.md:154]: Should We also mention that the headers pertaining to the selected algorithm is automatically added?

**Review by giridharvc7 — CHANGES_REQUESTED** [review]
Have added some comments, LGTM otherwise

**Pull request merged**